tools/create-cs.sh

```diff
@@ -0,0 +1,20 @@
+ #!/bin/bash
+tag="$1"
+
+[ $# -eq 0 ] && { echo "Usage: $0 <cincinnati-operator-registry image tag>"; exit 1; }
+
+DISCONNECTED_REGISTRY="${DISCONNECTED_REGISTRY:-quay.io/jottofar}"
```

Comment:
Do we have a canonical catalog location we can use as a default here?

Reply:
Also, doesn't have to be disconnected, so maybe just `IMAGE_REGISTRY`?

Reply:
changed in tools/create-catalog-source.sh